libs/opus/celt/celt_encoder.c

```diff
@@ -1429,7 +1429,7 @@ int celt_encode_with_ec(CELTEncoder * OPUS_RESTRICT st, const opus_val16 * pcm,
    celt_sig *prefilter_mem;
    opus_val16 *oldBandE, *oldLogE, *oldLogE2, *energyError;
    int shortBlocks=0;
-   int isTransient=0;
+   int isTransient=1;
```

Comment:
Does this have any effect?  A quick grep shows:
```
1432:   int isTransient=0;
1691:   isTransient = 0;
1699:      isTransient = transient_analysis(in, N+overlap, CC,
```

Reply:
Here is samples that I have encoded where *transient0 is without modification  and *transient1 is using what I posted. For some it is subtle but for example listen to how sharp the ride hits are. 
[NOFX-Shes_Gone.zip](https://github.com/jamulussoftware/jamulus/files/9447196/NOFX-Shes_Gone.zip)

Reply:
Thanks for the samples. I do see that the files have different checksums (but even running `opusenc` twice produces two different checksums), but I don't hear a difference and don't see one in the wave form either.
After @pljones's hint I'm not even sure if I should expect to hear one.

@38github Can you explain
1. what the flag does,
2. why it is safe to modify,
3. why it is supposed to change anything at all despite @pljones's remark that the change is effectively overwritten later in the code?

Also, has there been any public discussion with the opus devs?